openquake/server/urls.py

```diff
@@ -50,6 +50,8 @@
             re_path(r'^engine/aelo_changelog$',
                     views.aelo_changelog,
                     name="aelo_changelog"),
+            re_path(r'^v1/get_site_classes$', views.get_site_classes,
```

Comment:
Perhaps a longer name `v1/aelo_site_classes` would be more explicit?

Reply:
Done